service/handlers.go

```diff
@@ -366,13 +368,36 @@ func handleBranchLoad(c *Core, w *ResponseWriter, r *Request) {
 		w.Error(err)
 		return
 	}
+	reader := anyio.GzipReader(r.Body)
+	if format == "parquet" {
+		// This format requires a reader that implements io.ReaderAt and
+		// io.Seeker.  Copy the reader to a temporary file and use that.
+		//
+		// TODO: Add a way to disable this or limit file size.
+		f, err := os.CreateTemp("", "zed-serve-load-")
+		if err != nil {
+			w.Error(err)
+			return
+		}
+		defer f.Close()
+		defer os.Remove(f.Name())
+		if _, err := io.Copy(f, reader); err != nil {
```

Comment:
I would wrap this in a LimitedReader and pick a big number as a const and we can worry about configuring the limit later.  Maybe 10GB?

Reply:
I created #4236 for this. I think what we want is a way to disable the limit for Zui where this doesn't create a new threat (since anyone who can talk to the service can already fill the file system by loading data).